### DISCUSSION TITLE
Other versions of Perl 1

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#30] [Q&A] [answered]
Are there any other repositories with Perl 1 available?

**Comment by Stagyrite** [comment, marked as answer]
Yes, there are other versions of Perl 1, and other pages as well. Here's a quite complete list of them.

1. [GitHub/Stagyrite/Perl-1.0](https://github.com/stagyrite/Perl-1.0)
1. [GitHub/eltikia/Perl-1.0](https://github.com/eltikia/Perl-1.0)
1. [GitHub/Stagyrite/perl-1.0_16](https://github.com/Stagyrite/perl-1.0_16)
1. [GitHub/eltikia/perl-1.0_16](https://github.com/eltikia/perl-1.0_16)
1. [CPAN/perl-1.0_16](https://st.aticpan.org/source/RCLAMP/perl-1.0_16)
1. [GitHub/kaworu/perl1](https://github.com/kaworu/perl1)
1. [GitHub/eltikia/perl1](https://github.com/eltikia/perl1)
1. [ETLA/perl1](https://web.archive.org/web/20160913082457/https://www.etla.org/retroperl/perl1/)
1. [GitHub/AnaTofuZ/Perl-1.0](https://github.com/AnaTofuZ/Perl-1.0)
